src/test/scala/sparkz/core/api/http/PeersApiRouteSpec.scala

```diff
@@ -25,10 +27,15 @@ class PeersApiRouteSpec extends AnyFlatSpec
 
   private val addr = new InetSocketAddress("localhost", 8080)
   private val restApiSettings = RESTApiSettings(addr, None, None, 10 seconds)
+  private val restApiSettingsWithApiKey = RESTApiSettings(addr, Some("$2a$10$O5FAloC/gNuwpeoVKiV41.EcIlpOlk5hzsqYpleSmOEEKgj0j7BX6"), None, 10 seconds)
```

Comment:
Instead of using `Some("$2a$10$O5FAloC/gNuwpeoVKiV41.EcIlpOlk5hzsqYpleSmOEEKgj0j7BX6")`
can we use the source data and generate the credential from it?
so to understand what is under the hood of it.

Reply:
Done